firmware/use_arm_gcc.mk

```diff
@@ -1,30 +1,35 @@
 ifeq ($(TRGT),)
-	UNAME_S := $(shell uname -s)
-	UNAME_SP := $(shell uname -sp)
-$(info UNAME_SP:          $(UNAME_SP))
+	UNAME_SM := $(shell uname -sm)
+$(info UNAME_SM:          $(UNAME_SM))
+	KERNEL := $(firstword $(UNAME_SM))
+	MACHINE := $(lastword $(UNAME_SM))
 
-	ifeq ($(UNAME_S),Darwin)
+	ifeq ($(KERNEL),Darwin)
 		COMPILER_PLATFORM = arm-gnu-toolchain-11.3.rel1-darwin-x86_64-arm-none-eabi
-	else ifeq ($(UNAME_SP),Linux x86_64)
-		COMPILER_PLATFORM = arm-gnu-toolchain-11.3.rel1-x86_64-arm-none-eabi
+	else ifeq ($(KERNEL),Linux)
+		ifeq ($(MACHINE),x86_64)
+			COMPILER_PLATFORM = arm-gnu-toolchain-11.3.rel1-x86_64-arm-none-eabi
+		else ifeq ($(MACHINE),aarch64)
```

Comment:
These files don't actually exist in the toolchain dirs, so let's not support it here yet